doc/generators/nsxt.md

```diff
@@ -0,0 +1,54 @@
+# NSXT
+
+The nsx header designation has the following format:
+
+```
+target:: nsxt {section_name} {inet|inet6|mixed} section-id securitygroup securitygroupId
+section_name: specifies the name of the section all terms in this header apply to.
+inet: specifies that the resulting filter should only render IPv4 addresses.
+inet6: specifies that the resulting filter should only render IPv6 addresses.
+mixed: specifies that the resulting filter should render both IPv4 and IPv6 addresses.
+sectionId: specifies the Id for the section [optional]
+securitygroup: specifies that the appliedTo should be security group [optional]
+securitygroupId: specifies the Id of the security group [mandatory if securitygroup is given]
+(Required keywords option and verbatim are not supported in NSX)
+```
+
+
+## Nsxt
+The nsxt header designation has the following format:
+```
+target:: nsxt {section_name} {inet|inet6|mixed} section-id securitygroup securitygroupId
+```
+  * _section_name_: specifies the name of the dfw rule all terms in this header apply to. [mandatory field]
+  * _inet_: specifies the output should be for IPv4 only filters. This is the default format.
+  * _inet6_: specifies the output be for IPv6 only filters.
+  * _mixed_: specifies that the resulting filter should render both IPv4 and IPv6 addresses.
+  * _sectionId_: specifies the Id for the section [optional]
+  * _securitygroup_: specifies that the appliedTo should be security group [optional]
+  * _securitygroupId_: specifies the Id of the security group [mandatory if securitygroup is given]
+(Required keywords option and verbatim are not supported in NSX)
+## Term Format
```

Comment:
Shouldn't there be a blank line before and after a `##` paragraph in markdown?

Reply:
Not in this case - a blank line above will move it at the same level as Nsxt and a blank line below moves the content after at the same level as ##Term Format and I want it to be a subsection of it.